client/post-editor/media-modal/image-editor/image-editor-crop.jsx

```diff
@@ -75,13 +65,15 @@ const MediaModalImageEditorCrop = React.createClass( {
 	},
 
 	componentWillReceiveProps( newProps ) {
+		const { crop } = this.state;
+
 		if ( ! isEqual( this.props.bounds, newProps.bounds ) ) {
 			const imageWidth = newProps.bounds.rightBound - newProps.bounds.leftBound;
 			const imageHeight = newProps.bounds.bottomBound - newProps.bounds.topBound;
-			const newTop = newProps.bounds.topBound + newProps.crop.topRatio * imageHeight;
-			const newLeft = newProps.bounds.leftBound + newProps.crop.leftRatio * imageWidth;
-			const newBottom = newTop + newProps.crop.heightRatio * imageHeight;
-			const newRight = newLeft + newProps.crop.widthRatio * imageWidth;
+			const newTop = newProps.bounds.topBound + crop.topRatio * imageHeight;
```

Comment:
It looks like I'm getting a JS error here for `crop.topRatio` when rotating the image.